src/compiler/sys.ts

```diff
@@ -1626,14 +1625,17 @@ export let sys: System = (() => {
         };
         return nodeSystem;
 
-        /**
-         * `throwIfNoEntry` was added so recently that it's not in the node types.
-         * This helper encapsulates the mitigating usage of `any`.
-         * See https://github.com/nodejs/node/pull/33716
-         */
+        /** Calls fs.statSync, returning undefined if any errors are thrown */
         function statSync(path: string): import("fs").Stats | undefined {
-            // throwIfNoEntry will be ignored by older versions of node
-            return (_fs as any).statSync(path, { throwIfNoEntry: false });
+            // throwIfNoEntry is available in Node 14.17 and above, which matches our supported range.
+            try {

```

Comment:
Don’t you then need to set the stack size to make it better like it was there in getModifiedTime

Reply:
Yes, but the main point is to avoid doing that for Node and runtimes which properly implement `throwIfNoEntry`, but still not crash if the runtime doesn't, at the expense of that particular maybe being slower than it used to be.

Reply:
Of course, maybe none of this is worth it, since it's "just" a property write.